src/js/shot-generator/Character.js

```diff
@@ -293,14 +294,17 @@ const Character = React.memo(({
           z : rotation.z,
         }  
       } );}); 
-      console.log(mesh.name);
+      console.log(mesh);
+      //TODO(): Make a better way to identify built in model
```

Comment:
If isCustomModel returns false, it's a built-in model

Reply:
I'm not sure that isCustomModel returns correct answer. I'm loading horse.glb and it's showing that it's built-in. By looking into the logic of that method it just checks if path is not empty.

Reply:
You'd pass it the `sceneObject.model`